package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "conartist",
+  "name": "conartist-root",
```

Comment:
why is this renamed?

Reply:
You can't have a child package with the same name. For example, your web-elm folder package name is also conartist. You'll get an npm dependency error.
`ERR! Refusing to install conartist as a dependency of itself`

Reply:
wat since when